.github/workflows/codeql.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8  # v5.0.0
 
     - name: Initialize CodeQL
-      uses: github/codeql-action/init@64d10c13136e1c5bce3e5fbde8d4906eeaafc885  # v3.30.6
+      uses: github/codeql-action/init@e296a935590eb16afc0c0108289f68c87e2a89a5  # v4.30.7
```

Comment:
Weird versioning scheme. Did they suddenly release v4.30.7 skipping v4.0.0...v4.30.6?

I thought this was a typo of v3.30.7, but they also released v3.30.7

https://github.com/github/codeql-action/releases